Video.js

```diff
@@ -310,8 +328,24 @@ export default class Video extends Component {
       resizeMode: this.props.posterResizeMode || 'contain',
     };
 
+    //androidFullScreen property will only impact on android. It will be always false for iOS.
+    const videoStyle = this.state.androidFullScreen ? {
+      position: 'absolute',
```

Comment:
This `absolute` is relative to his father, so it does not work properly.
The video must move to the top of the `DOM` in order to position it `absolutely` with respect to the screen.

Only by modifying the styles I don't think we can get the full screen.

Reply:
@jorgemarrero
Thanks for your observation.

I have already considered the above scenario and created POC for the same. its worked fine for me. Will create a repo for the POC's and will share with you ASAP.

In react Native, position value as "Absolute" means - the element is positioned absolutely to its first positioned parent.

https://dzone.com/articles/css-position-relative-vs-position-absolute

Reply:
@jorgemarrero - Can you please approve this PR.